addons/lunch/static/src/scss/lunch_kanban.scss

```diff
@@ -0,0 +1,63 @@
+.o_list_view {
+    .o_data_row {
+        .o_text_overflow {
+            @include o-text-overflow(table-cell);
```

Comment:
This styling shouldn't be applied globally (all list views) once this specific module is installed.
Scope it (or fix it globally if it makes sense).

Reply:
Also in this case only set `display: table-cell` is necessary instead of applying the whole `o-text-overflow` mixin.